peeringdb_server/validators.py

```diff
@@ -438,9 +438,14 @@ def validate_social_media(value):
                     )
             elif service in ["instagram", "twitter", "tiktok", "facebook", "linkedin"]:
                 # validate username
-                regex = (
-                    r"^(-*)(?=.{4,32}$)(?![.\-])(?!.*[.]{2})[a-zA-Z0-9._\-]+(?<![.])$"
-                )
+                if service in ["twitter"]:
+                    regex = (
+                        r"^[a-zA-Z0-9_]{1,15}$"
```

Comment:
While X allows the screen name to be < 4 characters, their usernames still require a minimum length of 4. 

Tested on the platform to confirm: min username length is 4, max is 15

Other than that changes look good - lets run linting as well so the tests can run:

```
poetry run pre-commit run --all-files
```

In the project directory

Reply:
@netravnen want us to get the changes above?

Reply:
@grizz rebased on latest master branch